media-libs/ptex/files/ptex-2.1.28-20170213.patch

```diff
@@ -0,0 +1,173 @@
+diff -purN a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	2016-04-04 18:14:05.000000000 +0100
++++ b/CMakeLists.txt	2017-02-13 22:59:10.000000000 +0000
+@@ -16,7 +16,7 @@ else ()
+ endif ()
+ 
+ if (NOT WIN32)
+-    set(CMAKE_CXX_FLAGS "-std=c++98 -Wall -Wextra -pedantic")
++    set(CMAKE_CXX_FLAGS "-std=c++98 -Wall -Wextra -pedantic ${CMAKE_CXX_FLAGS}")
```

Comment:
I changed it to 11, because it's easy to modify a patch like that, and it seems to work. Will need to make sure it doesn't segfault under testing.